api/src/test/java/edu/cornell/mannlib/vitro/webapp/dynapi/RESTEndpointTest.java

```diff
@@ -19,31 +22,34 @@
 import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameter;
 import org.mockito.Mock;
 import org.mockito.MockedStatic;
+import org.mockito.MockitoAnnotations;
 import org.mockito.Spy;
-import org.mockito.junit.MockitoJUnitRunner;
 
 import edu.cornell.mannlib.vitro.webapp.dynapi.components.Action;
+import edu.cornell.mannlib.vitro.webapp.dynapi.components.HTTPMethod;
 import edu.cornell.mannlib.vitro.webapp.dynapi.components.OperationResult;
+import edu.cornell.mannlib.vitro.webapp.dynapi.components.RPC;
 import edu.cornell.mannlib.vitro.webapp.dynapi.components.Resource;
 import edu.cornell.mannlib.vitro.webapp.dynapi.components.ResourceKey;
 
-@RunWith(MockitoJUnitRunner.class)
+@RunWith(Parameterized.class)
```

Comment:
This change here potentially introduces a problem.
The `MockitoJUnitRunner.class` is what is supposed to process the annotations such as `@Mock` and `@Spy`.
I'm surprised there are no errors, maybe the mocks aren't needed?

For every `@Mock` you should in instead have a `Mockito.mock(XXX);` for creating a mock manually.
The spy is `Mockito.spy(XXX)`.

If I am wrong, then that is great and I can simplify my integration tests to use the annotations rather than directly calling the `Mockito.mock()` and `Mockito.spy()` methods.

I will investigate this some to determine if this is indeed a problem or is not the problem that I am thinking it is.

Reply:
Okay I see.
It seems that you introduced this:
```
MockitoAnnotations.openMocks(this);
```

Which processes the annotations, allowing for those annotations to be used when using a different runner.
I wasn't aware of this.
I can use this to improve the tests that I wrote.

Reply:
They error without that openMocks call.